pubspec.yaml

```diff
@@ -27,7 +28,6 @@ dependencies:
   intl: ^0.17.0
   path_provider: ^2.0.11
   pdfx: ^2.0.1+2
-  xml: ^5.3.1
```

Comment:
We need the xml package for parsing the RUB news items. The server provides the content just as XML.

Reply:
This was incorrectly removed, thanks for mentioning it. I have added the package again.